LayoutFrameworkBenchmark/Benchmarks/NKFrameLayoutKit/NKFrameLayoutKitView.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  NKFrameLayoutKitView.swift
+//  LayoutFrameworkBenchmark
+//
+//  Created by Nam Kennic on 6/24/18.
+//
+
+import UIKit
+import NKFrameLayoutKit
+
+/// A LinkedIn feed item that is implemented with NKFrameLayoutKit code.
+class NKFrameLayoutKitView: UIView, DataBinder {
```

Comment:
The layout is perfect, except you need to add a padding below the comment label and the Like label.